integration_test/cleanup_ignore_demo_test.dart

```diff
@@ -0,0 +1,282 @@
+        exploreTabFinder,
+        findsOneWidget,
+      ); // Ensure the "Explore" tab exists.
+      await tester.tap(exploreTabFinder); // Tap on the "Explore" tab.
+      await tester.pumpAndSettle(); // Wait for the UI to settle.
+
+      final visualSubTabFinder = find.text('Visual');
+      expect(
+        visualSubTabFinder,
+        findsOneWidget,
+      ); // Ensure the "Visual" sub-tab exists.
+      await tester.tap(visualSubTabFinder); // Tap on the "Visual" sub-tab.
+      await tester.pumpAndSettle(); // Wait for the UI to settle.
+
+      // Check that 'evaporation' is the selected variable.
+      final evaporationSelectedFinder = find
+          .text('evaporation')
+          .hitTestable(); // Check 'evaporation' is selected.
+      expect(
+        evaporationSelectedFinder,
+        findsOneWidget,
+      ); // Ensure 'evaporation' is selected.
+
+      await tester.tap(evaporationSelectedFinder); // Tap on the dropdown menu.
+      await tester.pumpAndSettle(); // Wait for the dropdown options to appear.
+
+      // Check that deleted variables are not in the dropdown options.
+      for (String variable in deletedVariables) {
+        final dropdownOptionFinder =
+            find.text(variable); // Find each variable in the dropdown options.
+        expect(
+          dropdownOptionFinder,
+          findsNothing,
+        ); // Ensure the deleted variable is not listed in the dropdown options.
+      }
+
+      // Navigate to the "Dataset" tab.
+      final datasetTabFinder = find.text(
+        'Dataset',
+      ); // Assuming the tab can be identified by its text 'Dataset'.
+      expect(
+        datasetTabFinder,
+        findsOneWidget,
+      ); // Ensure the "Dataset" tab exists.
+      await tester.tap(datasetTabFinder); // Tap on the "Dataset" tab.
+      await tester.pumpAndSettle(); // Wait for the UI to settle.
+      // Tap the right arrow button to go to the variable role selection.
+      await tester.tap(rightArrowFinder);
+      await tester.pumpAndSettle();
+      await tester.tap(rightArrowFinder);
+      await tester.pumpAndSettle();
+
+      // Find the "Ignore" buttons and click the first two.
+      final button2Finder = find.text('Ignore');
+      expect(
+        button2Finder,
+        findsWidgets,
+      ); // Check that multiple "Ignore" buttons exist
+
+      // Tap the first two "Ignore" buttons found
+      for (int i = 0; i < 2; i++) {
+        await tester.tap(button2Finder.at(i));
+        await tester.pumpAndSettle();
+      }
+
+      // Navigate to the "Transform" tab.
+      await tester.tap(transformTabFinder); // Tap the "Transform" tab.
+      await tester.pumpAndSettle(); // Wait for the UI to update.
+
+      // Navigate to the "Cleanup" sub-tab within the Transform tab.
+
+      await tester.tap(cleanupSubTabFinder); // Tap the "Cleanup" sub-tab.
+      await tester.pumpAndSettle(); // Wait for the UI to update.
+      // Tap on the single found "Ignore" chip.
```

Comment:
The IGNORE should still be the selected chip - can that be tested and fail if not. If it is then simply tap the BUILD